interface/patient_file/reminder/patient_reminders.php

```diff
@@ -246,10 +246,9 @@ function sel_patient() {
                     "a.sms_status, a.email_status, a.mail_status, b.fname, b.lname, b.hipaa_allowemail, b.hipaa_allowsms " .
         "FROM `patient_reminders` as a, `patient_data` as b " .
         "WHERE a.active='1' AND a.pid=b.pid " . $add_sql .
-        "ORDER BY " . add_escape_custom($sortby) . " " .
```

Comment:
https://stackoverflow.com/questions/5200051/is-using-mysqli-real-escape-string-enough-to-secure-my-query-string

Reply:
That still doesn't help the fact that you can't use sql binding for these elements :)
This is why there are specific functions to escape these different elements.

Reply:
I agree that add_escape_custom() is not being used properly here also. In applicable places for binding (in variables) we do prefer binding over add_escape_custom() if possible.

Reply:
To add to above. In variables, add_escape_custom() is just as safe as binding as long as there are quotations around the add_escape_custom() call. But again, we do prefer binding over add_escape_custom().